Doc/library/threading.rst

```diff
@@ -50,6 +52,9 @@ This module defines the following functions:
    :mod:`threading` module, a dummy thread object with limited functionality is
    returned.
 
+   .. deprecated:: 3.10
+      The function `currentThread` is an alias for this function.
```

Comment:
This markup looks like current_thread() is deprecated, whereas it it not. Please remove it.